client/src/pages/Login.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from "react";
+import axios from "axios";
+import { useHistory } from "react-router-dom";
+
+import LoginRegisterNavbar from "./LoginRegisterNavbar";
+
+import {
+  Button,
+  CssBaseline,
+  TextField,
+  Box,
+  Typography,
+  makeStyles,
+  Container
+} from "@material-ui/core";
+
+const useStyles = makeStyles(theme => ({
+  "@global": {
+    body: {
+      backgroundColor: theme.palette.common.white
+    }
+  },
+  paper: {
+    marginTop: theme.spacing(8),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center"
+  },
+  avatar: {
+    margin: theme.spacing(1),
+    backgroundColor: theme.palette.secondary.main
+  },
+  form: {
+    width: "100%",
+    marginTop: theme.spacing(1)
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+    borderRadius: "25px"
+  },
+  formInput: {
+    [`& fieldset`]: {
+      borderRadius: "25px"
+    }
+  },
+  error: {
+    color: "red",
+    display: "flex",
+    justifyContent: "center"
+  }
+}));
+
+const Login = () => {
+  const classes = useStyles();
+  const history = useHistory();
+  const [loginError, setLoginError] = useState("");
+
+  const onSubmitHandler = event => {
+    setLoginError("");
+    event.preventDefault();
+
+    axios
+      .post("http://localhost:4000/login", {
+        username: event.target.email.value,
+        password: event.target.password.value
+      })
+      .then(res => {
+        if (res.data.success) {
+          history.push("/dashboard");
+        }
+      })
+      .catch(() => setLoginError("Please check your email and password"));
+  };
+
+  const onClickHandler = () => {
+    setLoginError("");
+  };
+
+  return (
+    <div>
+      <LoginRegisterNavbar />
```

Comment:
I think we discussed just using a single Navbar across all pages. I believe Nicky added a prop called "loggedIn" that determines what will be displayed on the navbar. Since there are different displays for Login and Register, I think it's fine that to have a separate Login/Register Navbar, as long as the formatting/css is the same. Alternatively, could make the original Navbar component just have a state that keeps track of whether your on Login, Register, or already logged in so we don't have two navbar components that are nearly identical.

Reply:
ok I merged the navbars into one.